addon/templates/components/fm-widgets/checkbox.hbs

```diff
@@ -1,5 +1,4 @@
-{{input
-  type='checkbox'
-  checked=value
-  name=widgetAttrs.name
```

Comment:
Worried that this might cause issues... I think we need these args. Could we just convert them to like `@name=this.widgetAttrs.name`?

Reply:
`@name={{this.widgetAttrs.name}}`